resources/swiftlint.yml

```diff
@@ -1,4 +1,5 @@
 whitelist_rules:
+  - anyobject_protocol
```

Comment:
I don't think we need to update both `swiftlint.yml` and the `airbnb.swiftformat`. Since the rule links to the `anyObjectProtocol` SwiftFormat rule, I think we should just keep that one.

```suggestion
```